packages/server/src/stream/service.ts

```diff
@@ -135,4 +145,14 @@ export class StreamService extends Service {
       }
     }
   }
+
+  private async getChannel(conversationId: uuid) {
+    const convmaps = await this.mapping.convmap.listByConversationId(conversationId)
+    if (convmaps.length === 1) {
+      const tunnel = await this.mapping.tunnels.get(convmaps[0].tunnelId)
+      return this.channels.getById(tunnel!.channelId).name
+    } else {
+      return 'messaging'
+    }
```

Comment:
Here it might be nice to differentiate a multi-channel conversation from a conversation on an unknown channel. What do you think?

Reply:
Maybe but this isn't a change done in this PR so it's not necessary to change this here. But let's keep this in mind.